SB2018/2018-04-17.md

```diff
@@ -0,0 +1,21 @@
+---
+to: All Teams
+subject: Competition Details
+---
+
+Hi All!
+
+With less than a week to go with the competition, we'd like to give you the full details for turning up.
+
+# Getting to the competition
+
+I've attached a map of the event, it will take place in:
+
+> Building 42
+> University Rd
+> Southampton
+> SO17 1BJ
+
+Also on the map is the recommended car park. This car park is free on weekends.
+
+All of these details are mirrored on our website: https://sourcebots.co.uk/events/2018/competition/.
```

Comment:
If we're not actually putting the information on the website for this, the last sentence should be worded differetnyl

Reply:
as in, "You can see all information on the website:"

Reply:
All this information _is already_ on the website, besides parking map